tools/istio-iptables/pkg/constants/constants.go

```diff
@@ -21,19 +21,28 @@ const (
 	FILTER = "filter"
 )
 
+// Built-in iptables chains
+const (
+	INPUT       = "INPUT"
+	OUTPUT      = "OUTPUT"
+	FORWARD     = "FORWARD"
+	PREROUTING  = "PREROUTING"
+	POSTROUTING = "POSTROUTING"
+)
+
+var BuiltInChainsMap map[string]struct{}
```

Comment:
Build a constant map directly remove your `init()` function:
```suggestion
var BuiltInChainsMap = map[string]struct{}{
	INPUT: struct{}{},
	...
}
```